packages/terra-list/src/List.jsx

```diff
@@ -162,13 +162,17 @@ const List = ({
       case KeyCode.KEY_UP: {
         event.preventDefault();
         const previousIndex = currentIndex > 0 ? currentIndex - 1 : lastIndex;
-        listItems[previousIndex].focus();
+        if (listItems[previousIndex]) {
+          listItems[previousIndex].focus();
+        }
```

Comment:
These changes should be tested

Reply:
Yes have tested by creating the tar file with this change on terra-menu . For reference please check the screen recording attached in PR description for pre and post implementation .

Reply:
Sorry, bad wording on my part. I trust that this was tested before submitting.

What I meant was that we should write tests for these to test the condition and any potential edge cases. It can be a unit test or a wdio test.

Reply:
For this particular scenario not being able to create test :- These functionality can be hard because not being able to assert the focus invocation on null object ,which is making hard to write the test case to justify the changes made on List.jsx.

Reply:
Hmm I thought you could test the condition but that makes sense. If it's not possible, then the screen recording should be good enough. Thanks for trying!
+1